website/www/site/data/io_matrix.yaml

```diff
@@ -0,0 +1,377 @@
+            url: https://beam.apache.org/releases/javadoc/current/org/apache/beam/sdk/io/kinesis/KinesisIO.html
+      - transform: AmqpIO
+        description: AMQP 1.0 protocol using the Apache QPid Proton-J library
+        implementations:
+          - language: java
+            name: org.apache.beam.sdk.io.amqp.AmqpIO
+            url: https://beam.apache.org/releases/javadoc/current/org/apache/beam/sdk/io/amqp/AmqpIO.html
+      - transform: KafkaIO
+        description: Read and Write PTransforms for [Apache Kafka](https://kafka.apache.org/).
+        implementations:
+          - language: java
+            name: org.apache.beam.sdk.io.kafka.KafkaIO
+            url: https://beam.apache.org/releases/javadoc/current/org/apache/beam/sdk/io/kafka/KafkaIO.html
+          - language: py
+            name: apache_beam.io.external.kafka
+            url: https://beam.apache.org/releases/pydoc/current/apache_beam.io.external.kafka.html
+      - transform: PubSubIO
+        description: Read and Write PTransforms for [Google Cloud Pub/Sub](https://cloud.google.com/pubsub) streams.
+        implementations:
+          - language: java
+            name: org.apache.beam.sdk.io.gcp.pubsub.PubsubIO
+            url: https://beam.apache.org/releases/javadoc/current/org/apache/beam/sdk/io/gcp/pubsub/PubsubIO.html
+          - language: py
+            name: apache_beam.io.gcp.pubsub
+            url: https://beam.apache.org/releases/pydoc/current/apache_beam.io.gcp.pubsub.html
+          - language: py
+            name: apache_beam.io.external.gcp.pubsub
+            url: https://beam.apache.org/releases/pydoc/current/apache_beam.io.external.gcp.pubsub.html
+          - language: go
+            name: github.com/apache/beam/sdks/go/pkg/beam/io/pubsubio
+            url: https://godoc.org/github.com/apache/beam/sdks/go/pkg/beam/io/pubsubio
+      - transform: JmsIO
+        description: An unbounded source for [JMS](https://www.oracle.com/java/technologies/java-message-service.html) destinations (queues or topics).
+        implementations:
+          - language: java
+            name: org.apache.beam.sdk.io.jms.JmsIO
+            url: https://beam.apache.org/releases/javadoc/current/org/apache/beam/sdk/io/jms/JmsIO.html
+      - transform: MqttIO
+        description: An unbounded source for [MQTT](https://mqtt.org/) broker.
+        implementations:
+          - language: java
+            name: org.apache.beam.sdk.io.mqtt.MqttIO
+            url: https://beam.apache.org/releases/javadoc/current/org/apache/beam/sdk/io/mqtt/MqttIO.html
+      - transform: RabbitMqIO
+        description: A IO to publish or consume messages with a RabbitMQ broker.
+        implementations:
+          - language: java
+            name: org.apache.beam.sdk.io.rabbitmq.RabbitMqIO
+            url: https://github.com/apache/beam/blob/master/sdks/java/io/rabbitmq/src/main/java/org/apache/beam/sdk/io/rabbitmq/RabbitMqIO.java
```

Comment:
I wasn't sure why javadoc for RabbitMqIO and KuduIO wasn't available, so I just linked to the GitHub files instead. Any idea @pabloem ?

Reply:
it seems that javadoc export is deactivated for those IOs (see line 20 of `sdks/java/io/kudu/build.gradle` and `sdks/java/io/rabbitmq/build.gradle`) - @aromanenko-dev do you know if that's expected or should be fixed?

Reply:
@epicfaace could you file JIRA issues for these?

Reply:
https://issues.apache.org/jira/browse/BEAM-10098

Reply:
I've also made https://issues.apache.org/jira/browse/BEAM-10099

Reply:
@pabloem Hmm, I don't remember why it was done for only these 2 Java IOs... It was introduced by this PR - https://github.com/apache/beam/pull/7197
For now, I don't see we can't enable it back. 
Maybe @garrettjonesgoogle or @swegner, who worked on that PR, could add something?

Reply:
The PR #7197 essentially just switched the exporting of javadocs from an include list to an exclude list, and preserved the bit for all components that existed at that point in time. RabbitMqIO and KuduIO didn't have javadocs exported before my change either. Thus, the explanation for their lack of javadocs would be due to something before me. 

My suspicion is that since the exporting of javadocs was an include list before my change, it was easy to miss components, and those two were probably missed accidentally.

Reply:
I've opened https://github.com/apache/beam/pull/11837, as it does look like both IOs have some amount of javadocs defined in their headers.